lib/useful/append.js

```diff
@@ -0,0 +1,28 @@
+import Promise from 'bluebird';
+import sentry from '../config/sentry';
+
+function append(bot, msg, suffix) {
+  if (!suffix) {
+    bot.sendMessage(msg.channel, 'Usage: **`!!`** `text`');
+    return;
+  }
+
+  Promise.resolve(msg.channel.messages.getAll('author', msg.author))
+    .then(messages => {
+      messages.pop();
+      if (messages[0] === undefined) {
+        return;
+      }
+
+      let newCmd = messages.pop().content.trim().concat(suffix);
+      bot.sendMessage(msg.channel, newCmd);
```

Comment:
A few days ago I've added a patch where the bot can't accept commands from itself. Instead what I'd suggest is importing all the commands from `/lib/index.js` and recalling the correct command, passing `bot`, `msg`, and `suffix` that way.